--- p1_cml_div2_front_nobleed.spice
+++ p1_cml_div2_front_integrated_sinks.spice
@@ -34,0 +35,4 @@
+XQSEF1_M ef_p_m c_p1_comp_m e_scomp_ef1_m sub! npn13G2 Nx=3 mm_ok=1
+XRDEG_SEF1_M e_scomp_ef1_m VSS sub! rppd w=12.0u l=0.50u m=1 b=0 mm_ok=1
+XQSEF2_M ef_n_m c_p1_comp_m e_scomp_ef2_m sub! npn13G2 Nx=3 mm_ok=1
+XRDEG_SEF2_M e_scomp_ef2_m VSS sub! rppd w=12.0u l=0.50u m=1 b=0 mm_ok=1
@@ -73,0 +78,4 @@
+XQSEF1_S DIV2_P c_p1_comp_s e_scomp_ef1_s sub! npn13G2 Nx=3 mm_ok=1
+XRDEG_SEF1_S e_scomp_ef1_s VSS sub! rppd w=12.0u l=0.50u m=1 b=0 mm_ok=1
+XQSEF2_S DIV2_N c_p1_comp_s e_scomp_ef2_s sub! npn13G2 Nx=3 mm_ok=1
+XRDEG_SEF2_S e_scomp_ef2_s VSS sub! rppd w=12.0u l=0.50u m=1 b=0 mm_ok=1
