--- original baab
+++ candidate C45 RZ62
@@ -171,7 +171,7 @@
 XM12 out1 vbpc pc2 vdd sg13_hv_pmos w=384u l=4u ng=48 m=1 mm_ok=1
 XM20 out out1 vdd vdd sg13_hv_pmos w=192u l=1u ng=24 m=1 mm_ok=1
 XM21 out vbn vss vss sg13_hv_nmos w=96u l=1u ng=12 m=1 mm_ok=1
-XRZ out1 cz vss rppd w=1u l=6.2u m=1 b=0 mm_ok=1
-XCC cz out cap_cmim w=69u l=23u m=1 mm_ok=1
+XRZ out1 cz vss rppd w=1u l=62u m=1 b=0 mm_ok=1
+XCC cz out cap_cmim w=45u l=23u m=1 mm_ok=1
 .ends
 
