--- enabled74001-qualified-control
+++ prospective74101-temp100
@@ -7,7 +7,7 @@
 .include bgr.spice
 .include t2f.spice
 .option gmin=1e-15 abstol=1e-13 reltol=1e-4 vntol=1e-6 method=gear
-.temp 25
+.temp 100
 .global sub!
 Vsub sub! 0 dc 0
 Vdd vdd 0 dc 3.3
@@ -26,10 +26,10 @@
 set numdgt=15
 set wr_singlescale
 set wr_vecnames
-setseed 74001
+setseed 74101
 reset
 echo PHASE0_BEGIN
-set temp=25
+set temp=100
 op
 echo P0_BGR_BEFORE_BEGIN
 print @n.xbgr.xr1.nr1[nsmm_rsh]
