--- preservedOneOPcontinuation
+++ twoOPcontinuation
@@ -23168,6 +23168,7 @@
 alter Vh6 dc=0
 alter Vh7 dc=1.2
 op
+op
 wrdata qualification/@RUN@/code128.dat v(vref) v(iptat) v(vref_buf) v(vped) v(shp) v(isense) v(xt.icmp) v(xt.vth_soft) v(xt.vth_hard) i(vdda) i(vdd)
 alter Vs0 dc=1.2
 alter Vs1 dc=1.2
@@ -23185,6 +23186,7 @@
 alter Vh5 dc=1.2
 alter Vh6 dc=1.2
 alter Vh7 dc=0
+op
 op
 wrdata qualification/@RUN@/return127.dat v(vref) v(iptat) v(vref_buf) v(vped) v(shp) v(isense) v(xt.icmp) v(xt.vth_soft) v(xt.vth_hard) i(vdda) i(vdd)
 echo CHUNK_NON_BGR_AFTER_BEGIN
