--- originalNominal
+++ sameSettingsCurrentOutputs
@@ -17,6 +17,32 @@
 set wr_singlescale
 set wr_vecnames
 op
+echo EXTERNAL_PORT_QUERY_BEGIN
+print @n.xbgr.xm34.nsg13_hv_nmos[ide]
+print @n.xbgr.xm34.nsg13_hv_nmos[ige]
+print @n.xbgr.xm34.nsg13_hv_nmos[ise]
+print @n.xbgr.xm34.nsg13_hv_nmos[ibe]
+print @n.xbgr.xm34.nsg13_hv_nmos[ctype]
+print @n.xbgr.xm35.nsg13_hv_nmos[ide]
+print @n.xbgr.xm35.nsg13_hv_nmos[ige]
+print @n.xbgr.xm35.nsg13_hv_nmos[ise]
+print @n.xbgr.xm35.nsg13_hv_nmos[ibe]
+print @n.xbgr.xm35.nsg13_hv_nmos[ctype]
+print @n.xbgr.xm38.nsg13_hv_pmos[ide]
+print @n.xbgr.xm38.nsg13_hv_pmos[ige]
+print @n.xbgr.xm38.nsg13_hv_pmos[ise]
+print @n.xbgr.xm38.nsg13_hv_pmos[ibe]
+print @n.xbgr.xm38.nsg13_hv_pmos[ctype]
+print @n.xbgr.xr1.nr1[ibody]
+print @n.xbgr.xr16.nr1[ibody]
+print @q.xbgr.xq56.qnpn13g2[ic]
+print @q.xbgr.xq56.qnpn13g2[ib]
+print @q.xbgr.xq56.qnpn13g2[ie]
+print @q.xbgr.xq56.qnpn13g2[is]
+print @r.xbgr.xq56.rsub[i]
+print @c.xbgr.xq56.csub[i]
+echo EXTERNAL_PORT_QUERY_END
+wrdata external_ports.dat i(v.xbgr.vobs_xm34_d) i(v.xbgr.vobs_xm34_g) i(v.xbgr.vobs_xm34_s) i(v.xbgr.vobs_xm34_b) i(v.xbgr.vobs_xm35_d) i(v.xbgr.vobs_xm35_g) i(v.xbgr.vobs_xm35_s) i(v.xbgr.vobs_xm35_b) i(v.xbgr.vobs_xm38_d) i(v.xbgr.vobs_xm38_g) i(v.xbgr.vobs_xm38_s) i(v.xbgr.vobs_xm38_b) i(v.xbgr.vobs_xr1_1) i(v.xbgr.vobs_xr1_2) i(v.xbgr.vobs_xr1_bn) i(v.xbgr.vobs_xr16_1) i(v.xbgr.vobs_xr16_2) i(v.xbgr.vobs_xr16_bn) i(v.xbgr.vobs_xq56_c) i(v.xbgr.vobs_xq56_b) i(v.xbgr.vobs_xq56_e) i(v.xbgr.vobs_xq56_bn) v(xbgr.obs_xm34_d) v(xbgr.obs_xm34_g) v(xbgr.obs_xm34_s) v(xbgr.obs_xm34_b) v(xbgr.obs_xm35_d) v(xbgr.obs_xm35_g) v(xbgr.obs_xm35_s) v(xbgr.obs_xm35_b) v(xbgr.obs_xm38_d) v(xbgr.obs_xm38_g) v(xbgr.obs_xm38_s) v(xbgr.obs_xm38_b) v(xbgr.obs_xr1_1) v(xbgr.obs_xr1_2) v(xbgr.obs_xr1_bn) v(xbgr.obs_xr16_1) v(xbgr.obs_xr16_2) v(xbgr.obs_xr16_bn) v(xbgr.obs_xq56_c) v(xbgr.obs_xq56_b) v(xbgr.obs_xq56_e) v(xbgr.obs_xq56_bn)
 print @n.xbgr.xr1.nr1[nsmm_rsh]
 print @n.xbgr.xr1.nr1[nsmm_w]
 print @n.xbgr.xr1.nr1[nsmm_l]
